src/app/shared/components/list/list.component.ts

```diff
@@ -11,8 +11,8 @@ import { Orientation } from '../../models/orientation';
   styleUrls: ['./list.component.scss'],
 })
 export class ListComponent implements OnInit {
-  @Input({ required: true }) service!: IListService;
-  @Input() items: (typeof this.service.dataType)[] = [];
+  @Input() service!: IListService;
+  @Input() items: (typeof this['service']['dataType'])[] = [];
```

Comment:
Did the linter complain about formatting it as `this.service.dataType`? It didn't complain about that before which is weird.

Reply:
Nope, I was surprised on that too.